### PR TITLE
[Button] loading state focus ring alignment

### PR DESCRIPTION
**Pull request by QuintonJason**
## Description

- [x] updated the focus ring alignment for agenda items

## Screenshots

|  Before  |  After  |
|--------|--------|
|loadingbuttonagendaitemsbefore|loadingbuttonagendaitems|

## Testing in `sage-lib`

It is unseeable when the component is used in isolation.

## Testing in `kajabi-products`

1. (**LOW**) Update alignment of the focus ring when an agenda items is saved.
   - [ ] Coaching programs client view -> edit agenda item

### CODE REVIEW

**Review by joedakroub — APPROVED** [review]
Looking good! Nice update @QuintonJason. 👍🏼

File: packages/sage-assets/lib/stylesheets/components/_button.scss [hunk 1]
```diff
@@ -320,9 +320,12 @@ $-btn-loading-min-height: rem(36px);
 }
 
 .sage-btn--is-loading {
-  min-height: $-btn-loading-min-height;
   pointer-events: none;
 
+  &.sage-btn--icon-only-check::before {
```
Comment (monkeypox8) [inline packages/sage-assets/lib/stylesheets/components/_button.scss:325]: As a follow-up, I can see a case where we'd want to extend this pattern to other icons as well. I'd be in favor of using a class or data-attribute so we're not tied to this specific icon name.

**Review by monkeypox8 — APPROVED** [review]
👍🏼 LGTM, I wasn't able to see the "loading" animation when editing a coaching agenda item, but I also had several feature flags missing when testing with the bridge. In any case, the "check" icon was quickly removed from display.

**Comment by QuintonJason** [comment]
> 👍🏼 LGTM, I wasn't able to see the "loading" animation when editing a coaching agenda item, but I also had several feature flags missing when testing with the bridge. In any case, the "check" icon was quickly removed from display.

Will update: on the `kajabi-products` side when calling a button, add the followup prop `spinner_on_submit: "my aria label",` that activates it

**Pull request merged**